src/speech.cs

```diff
@@ -241,6 +259,38 @@ interface SFTranscriptionSegment : NSCopying, NSSecureCoding {
 
 		[Export ("alternativeSubstrings")]
 		string [] AlternativeSubstrings { get; }
+
+		[iOS (13, 0)]
+		[NullAllowed, Export ("voiceAnalytics")]
+		SFVoiceAnalytics VoiceAnalytics { get; }
+	}
+
+	[Mac (10,15, onlyOn64: true), iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	interface SFAcousticFeature : NSCopying, NSSecureCoding {
+
+		[Export ("acousticFeatureValuePerFrame", ArgumentSemantic.Copy)]
+		NSNumber[] AcousticFeatureValuePerFrame { get; }
+
+		[Export ("frameDuration")]
+		double FrameDuration { get; }
+	}
+
+	[Mac (10,15, onlyOn64: true), iOS (13,0)]
+	[BaseType (typeof (NSObject))]
+	interface SFVoiceAnalytics : NSCopying, NSSecureCoding {
```

Comment:
that looks like `[DisableDefaultCtor]` should be used here. Why ?
a. it's only getters (so why create your own instances); and
b. the only place it's exposed is a property with a getter only.

Reply:
So if it only met the first requirement and was being set somewhere else, would we want to still use the [DisableDefaultCtor]?

Reply:
A: yes, at least in most cases. A counter-example would be when values are read from a source, like time/counters, and be different for each instance you create (side effect).

note: don't resolve comments - since they become nearly invisible when we review the PR again (so they are easy to miss)

Reply:
Oops sorry!

Reply:
So should SFTranscriptionSegment also get the [DisableDefaultCtor] since it is only getters and only exposed with a getter?

Reply:
Yes, that hints us that it is not meant to be created by users since there is no way to set the object's properties nor you are able to use/set this object anywhere else.

Reply:
I also added it to SFSpeechRecognitionResult and SFTranscription